server.py

```diff
@@ -33,6 +35,27 @@ def get_pw(username):
 def index():
     return render_template('index.html', name='Cycle Project')
 
+@app.route('/predict', methods=['POST'])
+def predict():
+    if clf:
+        try:
+            json_ = request.json
+            query = pd.get_dummies(pd.DataFrame(json_))
+
+            # https://github.com/amirziai/sklearnflask/issues/3
+            # Thanks to @lorenzori
+
+            prediction = list(clf.predict(query))
+
+            return jsonify({'prediction': prediction})
+
+        except Exception as e:
+
+            return jsonify({'error': str(e), 'trace': traceback.format_exc()})
+    else:
+        print("train first")
```

Comment:
Can you instead run a system call to train the Model instead?

Reply:
You mean train the model in the server.py file by calling the function defined in model.py?